.github/workflows/extension-build.yml

```
@@ -0,0 +1,43 @@
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: leakr-extension
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20
+      - name: Cache pnpm modules
+        uses: actions/cache@v3
```

actionlint (1.7.7)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)